tests/functional/test_show.py

```diff
@@ -9,9 +9,8 @@ def test_show(script):
     Test end to end test for show command.
     """
     result = script.pip('show', 'pip')
-    lines = result.stdout.split('\n')
```

Comment:
Prefer `splitlines` over `split('\n')`
```
> stdout = 'aaaa\nbbb\n'
> stdout.split('\n')
['aaaa', 'bbb', '']
> stdout.splitlines()
['aaaa', 'bbb']
```
The tests were counting one more than usual.